### PR TITLE
Add UI Schema to implementations; 

### PR DESCRIPTION
**Pull request by elbakerino**
Added @ui-schema/ui-schema under ui generation and the corresponding live editor at editors.

### CODE REVIEW

File: implementations.md [hunk 1]
```diff
@@ -284,6 +285,7 @@ _None currently support draft-06 or later._
 -   [JSONEditor Online](https://jsoneditoronline.org/) - *View, edit, format, and validate JSON online* Support draft-4, draft-6, and draft-7.
 -   [Oxygen JSON Editor](https://www.oxygenxml.com/xml_editor/json.html) - *JSON editor with a variety of editing features and helper views. Support for validation and editing JSON Schema draft-4, draft-6, and draft-7. Validation and editing of JSON files based on JSON Schema.*
 -   [Stoplight Studio](https://stoplight.io/) - *JSON Schema IDE (text-based and GUI) with support for JSON/YAML linting, which can also be based on JSON Schema rules via Spectral. Support for draft-4, draft-6 and draft-7.*
+-   [UI Schema Live Editor](https://ui-schema.bemit.codes/examples/Simple-Demo) *Schema driven live editor with optional ui keywords, creates a form and validates the data with the given schema, supports 2019-09 / draft-08, -07, -06, -04 (incompatible `type=integer`)*
```
Comment (Relequestual) [inline implementations.md:288]:
```suggestion
```

**Comment by Relequestual** [comment]
It's really cool to see a UI generator which supports draft `2019-09`! Great work.
I'm super impressed by the stepped approach. NICE!
I'm a little (or a lot) jealous there's not good vue equivalent (at least that I know of).
I am a fan of THIS.

I modified your PR to remove adding the editor.
I don't feel the live preview is really a JSON Schema editor. It's really a text area with some auto complete.
I don't even list my own similar sandbox thing because it doesn't offer comparable features to an editor.
I hope you can understand my reasoning.

I've put this PR on hold. There are a few things I'd like to see addressed before adding you to the list.

- We ask libraries to always make clear the supported drafts of JSON Schema on their site. I can't see that, and I'd expect to see it on the home page. Given you support a good number, including `2019-09`, this is super important, especially if I'm going to direct people there.

- The "documentation" link on the home page in the body content takes me to a 404. The "home" link on the 404 page also leads to a 404. 😬 Minior I know.

---

Two things unrelated to this PR...

You should join our JSON Schema slack server. People are always asking about form generation, and it's a great place to hear about what's coming in the next version of JSON Schema, and even influence that outcome.

The way in which you've added keywords to JSON Schema is a great candidate for a new vocabulary.
I'd like to be able to guide you (and maybe a few others) through the process of formalising a vocabulary for UI and form generation. (I'm on the JSON Schema slack server under the same username.)

**Comment by Relequestual** [comment]
PR marked on hold as there isn't actually a change to the PR itself to be made.
/remind me to look at this again in 7 days.

**Comment by Relequestual** [comment]
/remind me to look at this again in 7 days.
PR marked as on hold as there isn't actually anything in the PR that needs to change now.

**Comment by elbakerino** [comment]
Thanks for the praise! There a still a few things missing, as just reaching v0.1.0, I'm always discovering new keywords in the spec. which i missed :D First version was only supporting what was available in the understanding book.

If someone likes to implement a similiar render engine in Vue, i would be glad to share typings and pure JS parts - there should be a lot not React dependent logic which could be bridged, e.g. shared validator plugin interfaces.

Alright with the editor, wasn't sure about it also.

The schema and related keyword support is the last point of [features](https://github.com/ui-schema/ui-schema#features) in github and on the homepage under [documentation > schema](https://ui-schema.bemit.codes/docs/schema#json-schema-keyword-support)

Thanks for the link info! Removed it, the page was deleted as the menu should be enough.

Regarding the vocabularies, would love to learn how to write them, already tried it at the beginning, [](https://github.com/ui-schema/ui-schema/blob/master/schema/2019-09/meta/view.json) (the `$id` link is also broken i realize now), and created some rather coplex [JSON Schema typing](https://github.com/ui-schema/ui-schema/blob/master/packages/ui-schema/src/JsonSchema.d.ts) to support nested schemas.

**Comment by Relequestual** [comment]
Documentation link on https://ui-schema.bemit.codes/ still leads to a 404 =]

**Pull request merged**